dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.connectionManagementJavaEE-1.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.connectionManagementJavaEE-1.0
+IBM-Provision-Capability:\
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.transaction-1.2)(osgi.identity=com.ibm.websphere.appserver.transaction-1.1)))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.connectionManagement-1.0))"
+IBM-Install-Policy: when-satisfied
+-features=com.ibm.websphere.appserver.javax.connector.internal-1.6; ibm.tolerates:=1.7
+-bundles=com.ibm.ws.jca.cm
+kind=ga
```

Comment:
best switch to noship. i know it's an autofeature but it will get flagged up when we release

Reply:
This is the Java EE autofeature. It would be a breaking change to make in nonship, in which case the existing jdbc features would become unusable for Java EE.  Hopefully you just misread this as being a Jakarta autofeature.  The change you requested cannot be made.

Reply:
OK, fair enough